### PR TITLE
Create Basic Stock Indexes API

### PR DESCRIPTION
**Pull request by hasan-yaman**

### CODE REVIEW

**Comment by MirageOwl** [comment]
A few reccomendations:
1. The variable names can be more specific like "stock_api_url" instead of just "url"
2. Maybe have a few comment lines or update the README as to how this API works and what the parameters do

**Comment by bekir96** [comment]
Can you move the whole logic here to `practice-app` under `/stock` endpoint. And instead of writing `stock_api_url` and `apikey` as inline variables, we can move them to `.env` or create a `config` file.

File: app.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+var express = require('express');
+var request = require('request');
+var bodyParser = require('body-parser');
```
Comment (mdumandag) [inline app.js]: body-parser is available on the express itself. We can get rid of that dependency. 
Just change
`app.use(bodyParser.json());` -> `app.use(express.json());`
`app.use(bodyParser.urlencoded({extended: false}));` -> `app.use(express.urlencoded({extended: false}));`

**Comment by hasan-yaman** [comment]
Thank you for the feedbacks guys. @bekir96  "Can you move the whole logic here to practice-app under /stock endpoint." I can't understand this line.

**Comment by mdumandag** [comment]
these files are not under the practice-app folder. We should revert this pr, move the files to the practice-app folder and then merge

**Pull request merged**